keps/sig-scheduling/4328-node-affinity-based-eviction/kep.yaml

```
@@ -0,0 +1,49 @@
+title: "Node Affinity Based Eviction" 
+kep-number: 4328
+authors:
+  - "@AxeZhan"
+owning-sig: sig-scheduling
+participating-sigs:
+  - sig-scheduling
+  - sig-node
+status: implementable
```

How about we aim to merge this as a provisional KEP, and then iterate?
```suggestion
status: provisional
```